.docker/entrypoint/spec/jekyll_commander_spec.rb

```diff
@@ -73,4 +74,53 @@
       end
     end
   end
+
+  describe '#execute :serve' do
+    describe '_site' do
+      data_dir = File.join(__dir__, '..', '..', '..', 'tests', 'full')
+      site_dir = File.join(data_dir, '_site')
+      @thread;
```

Comment:
Codacy found an issue: [Do not use semicolons to terminate expressions.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5780130)